### PR TITLE
New post: Customize how py.test collects tests from classes

### PR DESCRIPTION
**Pull request by nicoddemus**

### CODE REVIEW

File: content/customize_class_collection/contents.lr [hunk 1]
```diff
@@ -0,0 +1,81 @@
+name: Customize how py.test collects tests from classes
```
Comment (hackebrot) [inline content/customize_class_collection/contents.lr]:
```python
>>> titlecase('Customize how py.test collects tests from classes')
'Customize How py.test Collects Tests From Classes'
```

😁

File: content/customize_class_collection/contents.lr [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+You can implement your own collection rules by implementing the
+[pytest_pycollect_makeitem](http://pytest.org/latest/writing_plugins.html#_pytest.hookspec.pytest_pycollect_makeitem) hook.
+
+Simply add this code to a `conftest.py` file at the root of your tests' directory and
+that's it:
+
+
+```python
+def pytest_pycollect_makeitem(collector, name, obj):
+    if inspect.isclass(obj) and issubclass(obj, TestingUtils):
```
Comment (hackebrot) [inline content/customize_class_collection/contents.lr]: Can you please add the imports for inspect?

**Comment by nicoddemus** [comment]
Sorry for not answering before, I planned to apply the changes you requested later today when I got home. :stuck_out_tongue_closed_eyes:

**Comment by hackebrot** [comment]
Np! Have a free moment, so I figured to update the post myself. 😄

**Comment by nicoddemus** [comment]
OK then, thanks! :+1:

**Comment by hackebrot** [comment]
Can you please confirm that this is looking good to you? http://hackebrot.github.io/pytest-tricks/customize_class_collection/

**Comment by nicoddemus** [comment]
I guess so, thanks!

**Comment by hackebrot** [comment]
Cool! Tweet is out 👍

**Comment by hackebrot** [comment]
Thank you for writing this up! Pretty sure this is helpful to a bunch of users

**Comment by nicoddemus** [comment]
Absolutely.

I think it would be interesting to also advertise these posts on pytest-dev, to encourage users from pytest-dev to write new ones (specially if someone asks for help like in that post).

**Comment by hackebrot** [comment]
I agree.

**Pull request merged**